gson/src/main/java/com/google/gson/JsonObject.java

```diff
@@ -148,13 +148,23 @@ public Set<String> keySet() {
   /**
    * Returns the number of key/value pairs in the object.
    *
-   * @return the number of key/value pairs in the object.
+   * @return the number of key/value pairs in the objezct.
```

Comment:
Looks like you introduced a different typo by accident:
```suggestion
   * @return the number of key/value pairs in the object.
```

Also, there is no need to force-push. In case these changes are integrated, the commits will be squashed anyways.

Reply:
Appologies. resolved